src/components/note/NoteEditor.tsx

```
@@ -0,0 +1,144 @@
+import 'react-quill/dist/quill.snow.css';
+import { Controller, useForm } from 'react-hook-form';
+import { faFontAwesome } from '@fortawesome/free-solid-svg-icons/faFontAwesome';
+import { zodResolver } from '@hookform/resolvers/zod';
+import Button from '../common/button/Button';
+import Icon from '../common/icon/Icon';
+import '@/styles/textEditor.css';
+import { NoteSchema } from '@/lib/note-validation';
+import ErrorMessage from '../auth/ErrorMessage';
+
+export default function NoteEditor() {
+  const [title, setTitle] = useState('');
```

ESLint

[error] 'setTitle' is assigned a value but never used.

(no-unused-vars)